lib/std/Build.zig

```diff
@@ -518,6 +541,34 @@ const OrderedUserValue = union(enum) {
                 hasher.update(map_entry.name);
                 map_entry.value.hash(hasher);
             },
+            inline //
+            .lazy_path,
+            .lazy_path_list,
+            => |*lp_or_list, tag| {
```

Comment:
Instead, make the array list case `continue :sw list.items;` - this will generate less bloat since the impl can be shared.

Reply:
To be sure I understand, do you mean to wrap the single lazy path value in an arraylist, and pass that through to the arraylist branch? I will assume yes but let me know otherwise.

Reply:
e6392b0d1966e2f4d9b78d507028f85ecf71a9ad

Reply:
umm isn't the other way around better? pass `arraylist.items` to be handled the same as the slice case.

Reply:
Sorry, I'm not sure I understand, what slice case?

Reply:
point is don't use `inline` because it's unnecessary bloat. you should be able to share the logic at runtime.

Reply:
Currently all the union payloads are either single values, or ArrayLists, no slices. Although looking at this, it maybe makes more sense for these to be slices.

Reply:
Well, the changed code no longer uses inline as per your suggestion; I'm just confused on as to what you mean by "the other way around": lazy_path is a single lazy path, and lazy_path_list is an arraylist of lazy paths. There is no other case where the .items field of the latter could be passed to in order to share the branch.

Reply:
mmm is github glitching out for me? I definitely see two instances of it:

![image](https://github.com/user-attachments/assets/bcf82321-6574-4047-acfe-887979e8aed9)

![image](https://github.com/user-attachments/assets/30606f96-8e84-4acb-bcf1-a74843f77e47)

Reply:
Ah my apologies, I didn't realize there were other instances of inline switch cases that you were referring to, that's on me.

Reply:
e6392b0d1966e2f4d9b78d507028f85ecf71a9ad
Opted for separate functions in lieu of labelled switches, because the latter just wasn't a good fit in these cases (very ugly and complicated in comparison).

Reply:
sounds good!